File: FCUBEAPI/FreightMasters/Models/PartyMisLocationsList.cs
using Shared.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FreightMasters.Models
{
    public class PartyMisLocationsList
    {
        public List<PartyMisLocationModel> MisList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/partymislocationslist.ts
import { Paginationmodel } from "./paginationmodel";
import { Partymislocationmodel } from "./partymislocationsmodel";
export class Partymislocationlistmodel {
    misList: Partymislocationmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
